File: app/src/main/java/com/apps/asritha/fivecards/HomePage.java
package com.apps.asritha.fivecards;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import android.view.View;
import android.view.WindowManager;

public class HomePage extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.content_home_page);

        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
    }

    public void play(View v) {
        Intent i = new Intent(HomePage.this, MainActivity.class);
        startActivity(i);
    }

    public void rules(View v) {
        Intent i = new Intent(HomePage.this, Instructions.class);
        startActivity(i);
    }
}

File: app/src/main/java/com/apps/asritha/fivecards/HomePage.kt
package com.apps.asritha.fivecards

import android.content.Intent
import android.os.Bundle
import android.view.WindowManager
import androidx.activity.compose.setContent
import androidx.appcompat.app.AppCompatActivity
import com.apps.asritha.fivecards.ui.screens.GameStartScreen
import com.apps.asritha.fivecards.ui.theme.FiveCardsTheme

class HomePage : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)
        setContent {
            FiveCardsTheme {
                GameStartScreen(
                    onStartGame = { startActivity(Intent(this, MainActivity::class.java)) },
                    onHowToPlay = { startActivity(Intent(this, Instructions::class.java)) }
                )
            }
        }
    }
}
